object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionAnnotatedTypeFactory.java

```diff
@@ -75,6 +62,9 @@
 
   private final boolean useValueChecker;
 
+  /** The collection of built-in framework support for object construction checker */
+  private Collection<FrameworkSupport> frameworkSupports;
+
   // The list is copied from lombok.core.handlers.HandlerUtil. The list cannot be used from that
```

Comment:
Shouldn't this list have been removed, since it's also present in the lombok framework support?